lib/kana/convertor.rb

```diff
@@ -13,6 +13,8 @@ class << self
     def kana_html(site, html)
       return html unless @@mecab
 
+      html = html.gsub("\u00A0", " ")
```

Comment:
__[rubocop]__
* Use `tr` instead of `gsub`. [:ref](https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)

<!-- Comment by [SideCI](https://sideci.com) -->